api/helpers/indexes.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const winston = require('winston');
+
+const defaultLog = winston.loggers.get('default');
+
+/**
+ * Runs once in app.js. Builds text schema for:
+ * - Project: name
+ * - RecentActivity: headline
+ * - Document: documentFileName and displayName
+ * 
+ * @returns {void}
+ */
+module.exports.generateTextIndex = async () => {
```

Comment:
I have 2 concerns with the file directory / naming convention:
* Why is it in an api directory
* Why is it in a helper directory.
Seems like it should be either under database somewhere.   Maybe i'm missing something, any additional information why it is located here would be great.

Reply:
We've never discussed file/folder organization within LUP and I my guess is that "helpers" was meant as a catch-all directory. I see your point about this being maybe more on the "config" side.

In terms of moving outside of "api," I see your point, but I also think the indexes are inexorably tied to the API. For example, in indexes.js, I'm creating a text index for specific Schema fields(like Project name, Document displayName, etc.). So you can't have this text index without the API schema already existing.